lib/msf/core/rpc.rb

```diff
@@ -35,11 +36,11 @@ module JSON
     autoload :InternalError, 'msf/core/rpc/json/error'
     autoload :ServerError, 'msf/core/rpc/json/error'
     autoload :ApplicationServerError, 'msf/core/rpc/json/error'
+    autoload :MiddlewareCatchInternalError, 'msf/core/rpc/json/error'
```

Comment:
what does this do?

Reply:
Loads new exceptions into context, no longer need though so I'll remove it now

Reply:
Removed autoload